windows_containers/windows-containers-release-notes-7-x.adoc

```diff
@@ -28,6 +28,25 @@ For more information, see the Red Hat OpenShift Container Platform Life Cycle Po
 If you do not have this additional Red Hat subscription, you can use the Community Windows Machine Config Operator, a distribution that lacks official support.
 endif::openshift-origin[]
 
+[id="wmco-7-1-0"]
+== Release notes for Red Hat Windows Machine Config Operator 7.1.0
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 7.1.0 were released in link:https: //access.redhat.com/errata/RHBA-2023:NEED[RHBA-2023:NEED].
+
+[IMPORTANT]
```

Comment:
Should this be a [WARNING]? I guess my question is coming from another question: what happens if I am not on 4.12.3 and update? Do I lose data etc. I guess it depends on what level of importance this admonitions is. Wanted to point this out for consideration.

Reply:
@JoeAldinger The update fails/hangs if the cluster is lower than 4.12.3.